backend/de.metas.contracts/src/main/sql/postgresql/system/50-de.metas.contracts/5628050_sys_gh12598_IC_aggregation.sql

```diff
@@ -0,0 +1,5 @@
+-- 2022-02-27T23:39:09.748Z
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO C_AggregationItem (AD_Client_ID,AD_Column_ID,AD_Org_ID,C_Aggregation_ID,C_AggregationItem_ID,Created,CreatedBy,EntityType,IsActive,Type,Updated,UpdatedBy) VALUES (0,579362,0,540003,540054,TO_TIMESTAMP('2022-02-28 01:39:09','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.invoicecandidate','Y','COL',TO_TIMESTAMP('2022-02-28 01:39:09','YYYY-MM-DD HH24:MI:SS'),100)
```

Comment:
which new column does this item reference?

PS: C_Invoice_candidate.C_Flatrate_Term_ID, right?

Reply:
yep